nixos/modules/services/databases/postgresql.nix

```diff
@@ -428,7 +454,43 @@ in
           PostgreSQL superuser account to use for various operations. Internal since changing
           this value would lead to breakage while setting up databases.
         '';
-        };
+      };
+
+      upgradeFrom = mkOption {
+        type = types.nullOr (types.submodule ({ config, ... }: {
+          options = {
+            package = mkOption {
+              type = types.package;
```

Comment:
The default can be derived from stateVersion

Reply:
I think the default *old* version can be, but I am not so sure about the default *current*. What do you think?

Reply:
Defaulted.